src/main/java/frc/robot/commands/sequences/BaseAutonSequence.java

```diff
@@ -223,31 +50,30 @@ public BaseAutonSequence(
 
         addCommands(
             // Place preloaded game piece
-            goAndPlaceTop(initialPose, midPose1, midPose2, midpose3, placePose, elevatorPosition),
+            goAndPlaceTop(initialPose, midPose1, midPose2, midpose3, placeState),
```

Comment:
we should probably always reset our pose to the initialPose at the beginning of auton